src/main/webapp/login-script.js

```diff
@@ -4,10 +4,19 @@ async function signIn(googleUser) {
   const url = new URL("/login", window.location.origin);
   url.searchParams.set("token", token);
   url.searchParams.set("private", "false");
-  const response = await fetch(url, { method: "POST" });
-  const id = await response.json();
+  const loginResponse = await fetch(url, { method: "POST" });
+  const id = await loginResponse.json();
   if (id.verified) {
     // Successful sign-in.
+    const termList = await getTermList();
+    const select = document.getElementById("search-term");
+    const opt = document.createElement('option');
```

Comment:
I'd call this optionElement, little clearer that it's not optional (which is usually what opt means)